### PR TITLE
Update model with new base image version 28

### PR DESCRIPTION
**Pull request by QianqianNie**
Upgrade image version to 28

### CODE REVIEW

File: assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/spec.yaml [hunk 1]
```diff
@@ -24,4 +24,4 @@ tags:
   license: apache-2.0
   task: chat-completion
   author: mistralai
-version: 1
+version: 2
```
Comment (msp8955) [inline assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/spec.yaml:27]: Did you verify this model works with the current container?
This is new model added in Jan, can you double check if you haven't already

File: assets/models/system/tiiuae-falcon-40b-instruct/spec.yaml [hunk 2]
```diff
@@ -34,4 +34,4 @@ tags:
     ]    
   inference_supported_envs:
     - vllm
-version: 3
+version: 4
```
Comment (msp8955) [inline assets/models/system/tiiuae-falcon-40b-instruct/spec.yaml:37]: Please check with Sasidhar on the falcon instruct models, these are not released to the model catalog yet.

File: assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/spec.yaml (hunk 1 above)
Comment (QianqianNie) [reply assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/spec.yaml:27]: will do it separately, wanted to focus on image update for this pr

**Review by msp8955 — APPROVED** [review]

File: assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/spec.yaml (hunk 1 above)
Comment (msp8955) [reply assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/spec.yaml:27]: don't forget to update this one to 28, I don't see it updated yet in the blob.

**Review by vivram — APPROVED** [review]

**Pull request merged**